style.css

```diff
@@ -0,0 +1,235 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.wholepage {
+  width: 94%;
+  margin: 0 auto;
+}
+
+body {
+  font-family: sans-serif;
+  background-color: rgb(87, 5, 238);
+  line-height: 1.6;
+}
+
+#header {
+  padding: 24px 25px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: rgb(143, 142, 241);
+  border-radius: 10px;
+  position: relative;
+  width: 95%;
```

Comment:
Please, kindly change the width from 95% to 100% so it matches with the rest of the sections.